internal/prometheusbp/registry.go

```diff
@@ -1,4 +1,4 @@
-package internalv2compat
+package prometheusbp
```

Comment:
is this a new package?  It looks like it but I can't quite tell from the UI.

If so, I would strongly recommend renaming this since this PR has already shown us that this is going to cause conflicts.

Suggested names: `internalprom` `promint`

https://go.dev/blog/package-names

Reply:
it's not new, but since it's internal we can still rename it.

renamed to `prometheusbpint`.